measure/src/main/scala/org/apache/griffin/measure/configuration/enums/GriffinEnum.scala

```diff
@@ -0,0 +1,16 @@
+package org.apache.griffin.measure.configuration.enums
```

Comment:
There should be a license documentation in header of  each files.

Reply:
Made the changes and committed, you can refer this comment  2cd0b5d3d6bc6def5a7cac0092b69b95832df9b1. Please review.

Reply:
Typically, license documentation should be in the head of the file.  and developers can use `mvn scalastyle:check` command to check our code style.

 But so sorry, Griffin doesn't enable scala code style check by default now. 
I will enable scala code style check in `GRIFFIN-310`